specs/order_spec.rb

```diff
@@ -79,31 +81,44 @@
 end
 
 # TODO: change 'xdescribe' to 'describe' to run these tests
-xdescribe "Order Wave 2" do
+describe "Order Wave 2" do
   describe "Order.all" do
     it "Returns an array of all orders" do
-      # TODO: Your test code here!
-      # Useful checks might include:
-      #   - Order.all returns an array
-      #   - Everything in the array is an Order
-      #   - The number of orders is correct
-      #   - The ID and products of the first and last
-      #       orders match what's in the CSV file
-      # Feel free to split this into multiple tests if needed
+      require 'csv'
```

Comment:
This test is getting rather long - it would probably be worthwhile following the advice in the comment and splitting it into multiple tests.

Reply:
Also relevant here, the Ruby convention is to put`require` statements at the very top of your file. This is to make it easier to figure out how dependencies between files work out.

Ruby lets you put `require`s inside other code so you can do something like checking whether you're on Windows or Mac and including a different version of a gem as needed, but it's not recommended to put them deep in your files.